__tests__/app.test.js

```diff
@@ -104,6 +104,32 @@ describe("/api/articles/:article_id", () => {
       });
   });
 
+  test("PATCH 200: Respond with updated article with votes more than 0", () => {
+    const newArticleVote = {
+      inc_votes: 20,
+    };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(newArticleVote)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.votes).toBe(120);
```

Comment:
I think for this one, I'd like to see that you have actually returned everything! Usually, I would say to be concise, but this specifically says that it'll return the updated _article_, not just the votes :)